eldritch/monsters.py

```diff
@@ -427,6 +427,29 @@ def Zombie():
   )
 
 
+class EventMonster(Monster):
+  """A pseudo-monster used for events like Bank3 and Hospital2"""
+
+  def __init__(self, name, rating, pass_event, fail_event, toughness=1, attributes=None):
+    super().__init__(
+        name, "normal", "moon", {"evade": 0} | rating,
+        {"combat": 0, }, toughness, attributes or set()
+    )
+    self.pass_event = pass_event
+    self.fail_event = fail_event
+
+  def get_trigger(self, event, state):
+    if isinstance(event, events.PassCombatRound):
+      return self.pass_event
+
+    if not isinstance(event, events.CombatRound) or not event.check.is_done():
```

Comment:
For safety, also please make sure that event.check is not None before doing the rest of these checks. Should avoid some unpleasant surprises.